alfresco-webscripts-client/alfresco-webscripts-spring/src/main/java/eu/xenit/alfresco/webscripts/client/spring/http/InsecureSslHttpComponentsClientHttpRequestFactory.java

```diff
@@ -11,7 +12,9 @@
 public class InsecureSslHttpComponentsClientHttpRequestFactory extends HttpComponentsClientHttpRequestFactory {
 
     @SneakyThrows
-    public InsecureSslHttpComponentsClientHttpRequestFactory() {
+    public InsecureSslHttpComponentsClientHttpRequestFactory(HttpClient httpClient) {
+        super(httpClient);
```

Comment:
Wait, you are setting/passing an `HttpClient` here, but a few lines down, we create and set a new one with `this.setHttpClient(...)` ?

Reply:
Good point, I overlooked that.